src/js/angular/utils/local-storage-adapter.js

```diff
@@ -0,0 +1,59 @@
+angular
+    .module('graphdb.framework.utils.localstorageadapter', [])
+    .factory('LocalStorageAdapter', LocalStorageAdapter)
+    .value('LSKeys', {
+        'TABS_STATE': 'tabs-state',
+        'TABS_STATE_CURRENT_ID': 'tabs-state-currentid',
+        'TABS_STATE_MAXID': 'tabs-state-maxid',
+        'VIEW_MODE': 'viewMode',
+        'CLASS_HIERARCHY_HIDE_PREFIXES': 'classHierarchy-hidePrefixes',
```

Comment:
Ideally, these should be in their corresponding modules I guess.

Reply:
Yeah, although many of these keys are used in multiple modules, so I either should have them duplicated in every module or I should introduce a way to provide/plug them which I guess might lead to some timing/synchronization issues.